content/intro-to-storybook/angular/en/using-addons.md

```diff
@@ -30,15 +30,15 @@ Storybook is a wonderful [component-driven development environment](https://www.
 
 ### Using Controls to find edge cases
 
-With Controls QA Engineers, UI Engineers, or any other stakeholder can push the component to the limit! Let's consider the following example, what would happen to our `Task` if we added a **MASSIVE** string?
+With Controls, QA Engineers, UI Engineers, or any other stakeholder can push the component to the limit! Considering the following example, what would happen to our `Task` if we added a **MASSIVE** string?
 
 ![Oh no! The far right content is cut-off!](/intro-to-storybook/task-edge-case.png)
 
 That's not right! It looks like the text overflows beyond the bounds of the Task component.
 
-Controls allowed us to quickly verify different inputs to a component. In this case a long string. This reduces the work required to discover UI problems.
+Controls allowed us to quickly verify different inputs to a component. In this case, a long string and reduced the work required to discover UI problems.
```

Comment:
```suggestion
Controls allowed us to quickly verify different inputs to a component -- in this case, a long string -- and reduced the work required to discover UI problems.
```

Reply:
Fixed with the latest commits. Thanks for the clarification